airflow/providers/amazon/provider.yaml

```diff
@@ -41,8 +41,21 @@ versions:
   - 1.1.0
   - 1.0.0
 
-additional-dependencies:
+dependencies:
   - apache-airflow>=2.2.0
+  - boto3>=1.15.0
+  # watchtower 3 has been released end Jan and introduced breaking change across the board that might
+  # change logging behaviour:
+  # https://github.com/kislyuk/watchtower/blob/develop/Changes.rst#changes-for-v300-2022-01-26
+  # TODO: update to watchtower >3
+  - watchtower~=2.0.1
+  - jsonpath_ng>=1.5.3
+  - redshift_connector>=2.0.888
+  - sqlalchemy_redshift>=0.8.6
+  - pandas>=0.17.1
+  - mypy-boto3-rds>=1.21.0'
```

Comment:
Ah . good point. No firm plans yet, but this change should make it very easy. With this one we are able to completely separate out the maintenance of those (provider,yaml already has "additional-extras" field for that and this change makes it much better (you can specify multiple requirements per additional extra).. 

The next step will be to make sure that that the `[devel]` extra of each provider will be installed in the CI image of ours, even if the main "provider" does not have it. This should be very easy change after we merge this one.

Reply:
BTW. Good eye with the ' :facepalm:

Reply:
Resolved

Reply:
Yeah. Looked at devel extra separation  - it would be simple extension - we would have to move extra preparation and deduplication from "provider package" preparation to pre-commit, but that woudl be rather simple. 

Maybe you can create an issue /feature request for that ? I tihnk it woudl be nice way to limit some unnecessary dependencies for released providers.